target/linux/brcm63xx/base-files/lib/preinit/05_init_interfaces_brcm63xx

```diff
@@ -4,45 +4,33 @@
 
 set_preinit_iface() {
 	case "$(brcm63xx_board_name)" in
-	a4001n |\
-	a4001n1 |\
-	ar-5381u |\
-	ar-5387un |\
-	bcm963281tan |\
-	bcm96328avng |\
-	cpva642 |\
-	ct536_ct5621 |\
-	cvg834g |\
-	dgnd3700v1_dgnd3800b |\
-	dsl-2640b-b |\
-	dsl-274xb-c |\
-	dsl-274xb-f |\
-	dsl-275xb-d |\
-	fast2504n |\
-	fast2704v2 |\
-	hg553 |\
-	hg556a_* |\
-	hg520 |\
-	hg622 |\
-	neufbox6 |\
-	p870hw-51a_v2 |\
-	rta770bw |\
-	rta770w |\
-	spw303v |\
-	spw500v |\
-	v2110 |\
-	vr-3025un |\
-	vr-3025u |\
-	vr-3026e |\
-	wap-5813n)
-		ifname=eth0
-		;;
+	agpf-s0 |\
 	bcm96348gw |\
+	bcm96348gw-10 |\
 	bcm96348gw-11 |\
-	gw6000 |\
-	gw6200)
+	bcm96358vw |\
+	bcm96358vw2 |\
+	cpva502p |\
+	dg834gt |\
+	dmv-s0 |\
+	dsl-2650u |\
+	dv-201amr |\
+	dva-g3810bn |\
+	f5d7633 |\
+	fast2404 |\
+	hg520 |\
+	magic |\
+	neufbox4 |\
+	rg100a |\
+	rta1025W |\
+	td-w8900gb |\
+	usr9108 |\
+	v2500v_bb)
 		ifname=eth1
 		;;
+	*)
+		ifname=eth0
+		;;
 	esac
 }
 
```

Comment:
This sets the default ifname to eth0 and only sets it to eth1 for the specific devices that need it.
(will add this to the commit description)